src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme("unknown_enzyme","")
```

Comment:
That is what I feared that copying all the enzyme structures wouldn't be
very.efficient

But then the uid should also go into the xml for ease of use?

sent from my smartphone -- you may keep any typos you find.
On Apr 17, 2015 10:43 AM, "liangoaix" <notifications@github.com> wrote:

> In src/openms/source/METADATA/ProteinIdentification.cpp
> <https://github.com/OpenMS/OpenMS/pull/1288#discussion_r28578574>:
>
> > @@ -81,7 +81,8 @@ namespace OpenMS
> >      enzyme(UNKNOWN_ENZYME),
> >      missed_cleavages(0),
> >      peak_mass_tolerance(0.0),
> > -    precursor_tolerance(0.0)
> > +    precursor_tolerance(0.0),
> > +    digestion_enzyme("unknown_enzyme","")
>
> yes, it's unique. I think, instead of storing an enzyme, an enzyme name
> UID could be the solution..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenMS/OpenMS/pull/1288/files#r28578574>.
>